ReactiveCocoa/Swift/Property.swift

```diff
@@ -374,6 +374,7 @@ extension PropertyProtocol {
 /// its source outlives it too.
 public final class Property<Value>: PropertyProtocol {
 	private let sources: [AnyObject]
+	private let disposable: Disposable?
```

Comment:
The property relies on a captured variable to present the latest value, and there is an observer to the replayed producer to feed this variable. (The variable is also used to check for the existence of an initial value.)

If this `Property` deinitializes, this captured variable is no longer accessible (in terms of reads), so the observer can be disposed of.